cmake/SetupCompilerOptions.cmake

```diff
@@ -226,38 +226,53 @@ if(BUILD_SHARED_LIBS)
 endif()
 
 ################################
-# Enable C++11/14
+# C++ Standard
 ################################
 
 SET( CMAKE_CXX_EXTENSIONS OFF )
 SET( CMAKE_CXX_STANDARD_REQUIRED ON )
 
+set(BLT_CXX_STD "" CACHE STRING "Version of C++ standard")
+set_property(CACHE BLT_CXX_STD PROPERTY STRINGS c++98 c++11 c++17 c++17)
```

Comment:
`c++17` is here twice.  

Looks like the first one should be ``c++14`` ?

Reply:
Fixed thanks.